builder/frameworks/arduino/arduino-samd.py

```diff
@@ -170,6 +177,33 @@
             os.path.join(FRAMEWORK_DIR, "cores", BUILD_CORE, "api", "deprecated-avr-comp")
         ]
     )
+elif VENDOR_CORE == "clearcore":
+    env.Prepend(
+        CPPPATH=[
+            os.path.join(FRAMEWORK_DIR, "variants", "clearcore", "ThirdParty", "SAME53", "CMSIS", "Device", "Include"),
+        ]
+    )
+
+    env.Append(
+        CPPPATH=[
+            os.path.join(FRAMEWORK_DIR, "cores", "arduino", "api"),
+            os.path.join(FRAMEWORK_DIR, "cores", "arduino"),
+            os.path.join(FRAMEWORK_DIR, "variants", "clearcore"),
```

Comment:
Just to be clear, the only line I don't need is 191? Or also the two first arduino lines?

so, this would be the result:
```
CPPPATH=[
    os.path.join(FRAMEWORK_DIR, "cores", "arduino", "api"),
    os.path.join(FRAMEWORK_DIR, "cores", "arduino"),
    os.path.join(FRAMEWORK_DIR, "Teknic", "LwIP", "LwIP", "port", "include"),
    os.path.join(FRAMEWORK_DIR, "Teknic", "LwIP", "LwIP", "src", "include"),
    os.path.join(FRAMEWORK_DIR, "Teknic", "libClearCore", "inc")
],
```

Reply:
Correct, only `os.path.join(FRAMEWORK_DIR, "variants", "clearcore"),` is *not* needed.